packages/fxa-react/components/Header/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Header } from './index';
+import { LogoLockup } from '../LogoLockup';
+
+storiesOf('components/Header', module)
+  .add('basic', () => (
+    <Header left={<div>left content</div>} right={<div>right content</div>} />
+  ))
```

Comment:
Do we need basic anymore? I believe the header is always going to have the logo, no?

Reply:
Mmm you know what, cancel that. I see this is in fxa-react and could very well be used without a logo as the `left` prop.

Reply:
Though I would be curious to know the use case.. 🤔

Reply:
Yeah, it's purposefully agnostic so you can pass whatever React elements you need to in the prop, I'm not sure when we wouldn't use it with the logo but this way you can pass whatever you want in there.